test/lib/mayaUsd/fileio/testCacheToUsd.py

```diff
@@ -45,6 +45,45 @@
 
 import os
 
+def createMayaRefPrimSiblingCache(testCase, cacheParent, cacheParentPathStr):
```

Comment:
Factored out differences between sibling cache and variant set cache workflows into functions passed to the test engine function.